tests/test_endpoints.py

```diff
@@ -18,44 +18,56 @@ def test_ping(nhsd_apim_proxy_url):
 def test_wait_for_ping(nhsd_apim_proxy_url):
     retries = 0
     resp = requests.get(f"{nhsd_apim_proxy_url}/_ping")
-    deployed_commitId = resp.json().get("commitId")
+    deployed_commit_id = resp.json().get("commitId")
 
-    while (deployed_commitId != getenv('SOURCE_COMMIT_ID')
-            and retries <= 30
-            and resp.status_code == 200):
+    while (deployed_commit_id != getenv('SOURCE_COMMIT_ID')
+           and retries <= 30
+           and resp.status_code == 200):
         resp = requests.get(f"{nhsd_apim_proxy_url}/_ping")
-        deployed_commitId = resp.json().get("commitId")
+        deployed_commit_id = resp.json().get("commitId")
         retries += 1
 
     if resp.status_code != 200:
         pytest.fail(f"Status code {resp.status_code}, expecting 200")
     elif retries >= 30:
         pytest.fail("Timeout Error - max retries")
 
-    assert deployed_commitId == getenv('SOURCE_COMMIT_ID')
+    assert deployed_commit_id == getenv('SOURCE_COMMIT_ID')
 
 
 @pytest.mark.smoketest
 def test_status(nhsd_apim_proxy_url, status_endpoint_auth_headers):
     resp = requests.get(
         f"{nhsd_apim_proxy_url}/_status", headers=status_endpoint_auth_headers
     )
+    resp_content = resp.json()
+    healthcheck_resp = resp.json().get("checks").get("healthcheck")
```

Comment:
```suggestion
    healthcheck_resp = resp_content.get("checks").get("healthcheck")
```
Minor, I think .json is doing some decoding so no need to perform it again